probe-rs/src/architecture/arm/core/armv8m.rs

```diff
@@ -289,9 +303,9 @@ impl<'probe> CoreInterface for Armv8m<'probe> {
 
     fn registers(&self) -> &'static RegisterFile {
         if self.state.fp_present {
-            &CORTEX_M_WITH_FP_REGS
+            &CORTEX_V8_M_COMMON_REGS
         } else {
-            &CORTEX_M_COMMON_REGS
+            &CORTEX_V8_M_WITH_FP_REGS
         }
```

Comment:
I'm afraid the register handling has changed (in #1620), so you'll have to update this. Sorry about that :grimacing:

Reply:
No worries.